scripts/vagrant/build.sh

```diff
@@ -33,14 +33,15 @@ echo DISTRIB_CODENAME: $DISTRIB_CODENAME
 echo DISTRIB_DESCRIPTION: $DISTRIB_DESCRIPTION
 
 if [ -f /etc/lsb-release ]; then
-    apt-get install -y gcc g++ make libcunit1-dev libaio-dev libssl-dev
+    apt-get install -y gcc g++ git make libcunit1-dev libaio-dev libssl-dev
 elif [ -f /etc/redhat-release ]; then
     # Add EPEL repository for CUnit-devel
-    yum --enablerepo=extras install epel-release
-    yum install -y gcc gcc-c++ CUnit-devel libaio-devel openssl-devel
+    yum --enablerepo=extras install -y epel-release
+    yum install -y gcc gcc-c++ git CUnit-devel libaio-devel openssl-devel
 fi
 
 cd $SPDK_DIR
+git submodule update --init
```

Comment:
Hi kriw,

Thanks for your patch!

I would prefer to leave anything related to submodule update in a separate patch, for two reasons:

1) this patch then focuses just on CUnit-devel on CentOS 7 problem
2) submodule update in the VM may not work as-is like this, especially if the host OS cloned using SSH;  I would prefer that we modify Vagrantfile to check that the submodules were updated instead

Thanks,

-Jim